src/api/app/views/group/index.xml.builder

```diff
@@ -1,5 +1,5 @@
 xml.directory(count: @list.length) do
-  @list.each do |group|
+  @list.order(title: :asc).each do |group|
```

Comment:
`@list` could be an ActiveRecord::Relation or an Array... 
Better to order the list in the controller instead of in the view